transcom/mymove
# [ MB-15435 ] Generating UUIDs to avoid collisions

## [Jira ticket](https://dp3.atlassian.net/browse/MB-15435)

## Summary

This was first addressed in #11033, but while reviewing #11187 I ran into a UUID
collision again. I've posted in [#g-testing][slack]🔒. I'm looking for some
feedback here to see if this fix works for all the other `devseed` scenarios.
I'll be doing further testing on Monday and will document on this PR what I
find. I didn't include this as part of #11187 so that I don't hold up that work.

[slack]: https://ustcdp3.slack.com/archives/CNTE04R5Z/p1691793822591899

## Verification Steps for Reviewers

Run the E2E Populate command in the repository and have it exist successfully.

### How to test

1. Run `make db_dev_e2e_populate`.
1. Have a successful run of that command and verify that `devseed` data
   scenarios aren't broken.
